dashboard/lti_new.py

```diff
@@ -181,7 +183,6 @@ def get_cache_config():
     cache_lifetime = cache_ttl if cache_ttl else 7200
     return CacheConfig(is_dummy_cache, launch_data_storage, cache_lifetime)
 
-
```

Comment:
Nit-pick: PEP recommends two blank lines between top-level functions/classes: https://peps.python.org/pep-0008/#blank-lines

Reply:
Added the empty line.